pkg/cmd/roachtest/github_test.go

```diff
@@ -103,6 +105,27 @@ func TestShouldPost(t *testing.T) {
 	}
 }
 
+func TestGenerateHelpCommand(t *testing.T) {
+	vo := vm.DefaultCreateOpts()
+	vmOpts := &vo
+
+	reg := makeTestRegistry(spec.GCE, "", "", false, false)
+	clusterSpec := reg.MakeClusterSpec(1, spec.Arch("arm64"))
+	testClusterImpl := &clusterImpl{name: "foo-cluster", spec: clusterSpec, arch: vm.ArchAMD64}
+
+	github := &githubIssues{
+		vmCreateOpts: vmOpts,
+		cluster:      testClusterImpl,
+		teamLoader:   validTeamsFn,
+	}
+
+	r := &issues.Renderer{}
+	generateHelpCommand(github)(r)
+
+	// TODO (BEFORE MERGE): time.Now() in tests == no bueno. could do regex, but want something like utils.TimeNow() that cc has
+	echotest.Require(t, strings.Replace(r.String(), "\n", "", -1), filepath.Join("testdata", "help_command.txt"))
```

Comment:
Why remove the newlines? Looks weird in the output, don't we want the actual output?

Reply:
hello - one benefit of removing newlines is that comparing actual vs expected outputs on error becomes a bit easer in Goland; however, yes the downside is that it looks weird in the output~ i shall fix this then.

Reply:
hi again - i just remembered: it seems like echotest stops reading from the "expected" output file once it reads a newline

so something like

> echo
> ----
> See: stuff here
> 
> See: more stuff here

cuts off after "See: stuff here"

Reply:
Have you tried running with `-rewrite`? There is a syntax that allows for multi-line. I think it's

```
echo
----
----
foo

bar
----
----
```

but better let `-rewrite` do it properly.

Reply:
ah! thank you for pointing this out